### DISCUSSION TITLE
Formula for Dummies

### DISCUSSION THREAD

**Discussion by Janita-S** [RafaelGB/obsidian-db-folder#1036] [Q&A]
Hi there?
What language do I use in the Formula Property?
I want to convert Calories to Kilojoules buy multiplying by 4.18
So....
Column1 = Column2 x 4.18
I read the "Formulas" page of the documentation and I have no clue 
I hope somone can help.
<img width="975" alt="Screenshot 2024-03-03 at 7 40 44 am" src="https://github.com/RafaelGB/obsidian-db-folder/assets/161977202/1e58b651-2542-4f40-9cda-588573d6b276">

**Comment by TheVasalex** [comment]
The language is called Javascript. This is what you should put in the formula:
${row["cals-100g"] * 4.18}

**Comment by Janita-S** [comment]
Thanks so much.
I really appreciate it.
: )
